frontend/src/pages/landing/Landing.tsx

```diff
@@ -198,18 +204,40 @@ export const Landing = () => {
         title: "Vernietigingslijsten",
         fieldsets: STATUSES,
         objectLists: objectLists,
+        toolbarProps: {
+          items: [
+            // TODO: SORT
```

Comment:
Is there a particular reason these are all TODO? Is that not in the scope of this PR?